library_generation/test/integration_tests.py

```diff
@@ -40,21 +41,21 @@
 repo_prefix = "https://github.com/googleapis"
 output_dir = shell_call("get_output_folder")
 # this map tells which branch of each repo should we use for our diff tests
-committish_map = {
+commitish_map = {
```

Comment:
Fun fact: for a long time I thought it _only_ was `committish` (two `t`s). 
I happens to be an alternative convenience term coming from `commit-ish`.


The [git glossary](https://git-scm.com/docs/gitglossary#Documentation/gitglossary.txt-aiddefcommit-ishacommit-ishalsocommittish) offers either `commit-ish` or `committish`. I always thought this would be too much to start a campaign to update all config yamls and its definition.

Reply:
Thanks Diego! That's good to know! Yeah I agree we should follow the naming conventions. Maybe an enhancement in the backlog.